README.md

```diff
@@ -759,16 +759,47 @@ available](http://enable-cors.org/server.html) for popular webservers
 and most CDNs should have no trouble turning CORS on for your account.
 
 
-## Known Issues
+## Known Issues and Workarounds
 Issues that are currenty known. If you want to
 help find a solution that would be appreciated!
 
 ### Fragmented MP4 Support
 Edge has native support for HLS but only in the MPEG2-TS container. If
-you attempt to play an HLS stream with fragmented MP4 segments, Edge
-will stall. Fragmented MP4s are only supported on browser that have
+you attempt to play an HLS stream with fragmented MP4 segments (without
+[overriding native playback](#overridenative)), Edge will stall.
+Fragmented MP4s are only supported on browsers that have
 [Media Source Extensions](http://caniuse.com/#feat=mediasource) available.
 
+### Assets with an Audio-Only Rate Get Stuck in Audio-Only
+Some assets which have an audio-only rate and the lowest-bandwidth
+audio + video rate isn't that low get stuck in audio-only mode. This is
+because the initial bandwidth calculation thinks it there's insufficient
+bandwidth for selecting the lowest-quality audio+video playlist, so it picks
+the only-audio one, which unfortunately locks it to being audio-only forever,
+preventing a switch to the audio+video playlist when it gets a better
+estimation of bandwidth.
+
+Until we've implemented a full fix, it is recommended to set the
+[`enableLowInitialPlaylist` option](#enablelowinitialplaylist) for any assets
+that include an audio-only rate; it should always select the lowest-bandwidth
+audio+video playlist for its first playlist.
+
+It's also worth mentioning that Apple no longer recommends having an
```

Comment:
Would be great if we can link to apple's docs on this

Reply:
Good idea. It no longer even hints at anything audio-only. Excellent.

It doesn't seem to like my markdown link though, do you know why that is?

Reply:
Ah, seems to be working in the PR preview, but generally, it looks like:
```md
[text](http://example.com)
[text][link]
[text link][]

[link]: http://example.com
[text link]: http://example.com
```
You want the 3rd example with the extra `[]`